requirements/pyproject.toml

```diff
@@ -28,7 +31,7 @@ ckantoolkit = "0.0.3"
 cryptography = "2.7"
 pika = ">=1.1.0"
 pyOpenSSL = "18.0.0"
-redis = ">=3.3.0"
+redis = "2.10.6"
```

Comment:
Why the downgrade? Seems like we should be opening a bug for this.

Reply:
@adborden the CKAN team [make some changes](https://github.com/ckan/ckanext-harvest/commit/9c4b0b22d6ded70ec962e2fbccf69e61210ca5c9) and inform that: `On CKAN 2.7 and 2.8: will use the core version (2.*)` 
If we don't use that version we will have issues running workers